spateo/plotting/static/pairwise.py

```diff
@@ -0,0 +1,185 @@
+"""
+Plots for pairwise comparison analyses between groups of samples and across different conditions (e.g. biological
+conditions or using different clustering parameters).
+
+These functions build on the Label class- see spateo.utils.Label for documentation
+"""
+from typing import List, Tuple, Union
+
+import matplotlib as mpl
+import matplotlib.patheffects as PathEffects
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy
+from anndata import AnnData
+from matplotlib.collections import PolyCollection
+from matplotlib.ticker import StrMethodFormatter
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+from ...utils import Label, interlabel_connections
+
+plt.rcParams["figure.dpi"] = 300
+
+
+def plot_connections(
+    label: Label,
+    weights_matrix: Union[scipy.sparse.csr_matrix, np.ndarray],
+    ax: Union[None, mpl.axes.Axes] = None,
+    figsize: tuple = (8, 8),
+    zero_self_connections: bool = True,
+    normalize_by_self_connections: bool = False,
+    shapes_style: bool = True,
+    max_scale: float = 0.46,
+    colormap: Union[str, dict, "mpl.colormap"] = "Spectral",
+    title_str="connection strengths between types",
+    title_fontsize: float = 12,
+    label_fontsize: float = 12,
+    verbose: bool = True,
```

Comment:
would be nice to add the following arguments: 

```
    save_show_or_return: Literal["save", "show", "return", "both", "all"] = "show",
    save_kwargs: Optional[dict] = None,
```
to be consistent with other functions

Reply:
also we have a system level setup for the verbose in the logger. may need to integrate with that 
```Set the logging level. Various logging level can be setted according to your needs:

DEBUG: useful for dynamo development, show all logging information, including those debugging information

INFO: useful for most dynamo users, show detailed dynamo running information

WARNING: show only warning information

ERROR: show only exception or error information

CRITICAL: show only critical information

%matplotlib inline
from dynamo.dynamo_logger import main_info, LoggerManager
LoggerManager.main_logger.setLevel(LoggerManager.INFO)```

Reply:
(here the dynamo logger is used as an example)